Add icon toolbars in capture view.

pierricgimmig opened the pull request:
- Add stb_image to load png files
- Add icons from Google's https://material.io/resources/icons/
- Add post-build steps to package icons folder next to executable
- Add common actions, track filter, search filter and process info
  toolbars in capture view

pierricgimmig commented:
![icon_toolbar](https://user-images.githubusercontent.com/3687222/83605128-3b48eb00-a52c-11ea-8d35-1304860940a5.png)

beckerhe commented:
Short question upfront regarding priorities: Are you planning on merging this into 1.48?

pierricgimmig commented:
> Are you planning on merging this into 1.48?

Ideally yes, but I understand it's a relatively big change. I submitted into master in case there are issues with the PR.

beckerhe commented on OrbitGl/ImGuiOrbit.cpp:
```diff
@@ -170,6 +171,51 @@ void Orbit_ImGui_CharCallback(GlCanvas* a_GlCanvas, unsigned int c) {
+  int image_height = 0;
+  unsigned char* image_data =
+      stbi_load(filename, &image_width, &image_height, NULL, 4);
+  if (image_data == NULL) return false;
```
nullptr

ronaldfw commented:
> > Are you planning on merging this into 1.48?
> 
> Ideally yes, but I understand it's a relatively big change. I submitted into master in case there are issues with the PR.

I consider this a fix for b/156245865. Agree that it's a larger fix that I would normally not consider purely a "bugfix", but the quality delta due to this change is quite large IMO, so I think it's worth the risk.

ronaldfw commented on OrbitGl/ImGuiOrbit.h:
```diff
@@ -40,6 +40,9 @@ IMGUI_API void Orbit_ImGui_KeyCallback(GlCanvas* a_Canvas, int key, bool down);
 void SetupImGuiStyle(bool bStyleDark_, float alpha_);
+bool LoadTextureFromFile(const char* filename, uint32_t* out_texture,
+                         int* out_width, int* out_height);
+uint32_t LoadTextureFromFile(const char* filename);
```
Document possible return values? (In particular, 0 means error here. Something like: "Returns the texture id generated by OpenGL. In case of an error, returns 0.")

ronaldfw commented on OrbitGl/ImGuiOrbit.cpp:
```diff
@@ -170,6 +171,51 @@ void Orbit_ImGui_CharCallback(GlCanvas* a_GlCanvas, unsigned int c) {
+      stbi_load(filename, &image_width, &image_height, NULL, 4);
+  if (image_data == NULL) return false;
+
+  // Create a OpenGL texture identifier
```
Super-nit

```suggestion
  // Create an OpenGL texture identifier
```

pierricgimmig replied:
Super-done.

ronaldfw commented on third_party/stb/LICENSE:
```diff
@@ -0,0 +1,37 @@
+This software is available under 2 licenses -- choose whichever you prefer.
```
We should only include the license we actually use (MIT?).

pierricgimmig commented:
Hi @beckerhe , thanks for the feedback!  I added the license for the icons, and modified the stb license to only show the MIT section.  Concerning the conan package, I had actually checked and all I found was a java port of the library.  Good to know that it is indeed available for C++.  Using the conan version will have to be done in an other PR though due to time constraints for release.

pierricgimmig commented on OrbitQt/orbitmainwindow.cpp:
```diff
@@ -430,6 +430,10 @@ void OrbitMainWindow::OnReceiveMessage(const std::string& a_Message) {
     ui->MainTabWidget->setCurrentWidget(ui->CaptureTab);
+  } else if (absl::StartsWith(a_Message, "opencapture")) {
+    on_actionOpen_Capture_triggered();
+  }else if (absl::StartsWith(a_Message, "savecapture")) {
```
Done

pierricgimmig commented on OrbitGl/CaptureWindow.cpp:
```diff
@@ -936,31 +899,157 @@ void CaptureWindow::RenderHelpUi() {
+  // Action Toolbar.
+  ImGui::Begin("Toolbar", &m_DrawHelp, ImVec2(0, 0), 1.f,
                     ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoResize |
                         ImGuiWindowFlags_NoMove |
```
Done
